pybotvac/account.py

```diff
@@ -218,7 +218,11 @@ def refresh_robots(self):
 
         self.refresh_persistent_maps()
         for robot in self._robots:
-            robot.has_persistent_maps = robot.serial in self._persistent_maps
+            if ("persistent_maps" in robot.traits
```

Comment:
I believe this first check is redundant, and that simply setting
`robot.has_persistent_maps = self._persistent_maps.get(robot.serial, [])) > 0`
should suffice?

Reply:
Yes, I think so too. I noticed that the map lists are empty if "persistent_maps" is not a trait and the other way around. Not sure if that always the case, so therefore I added the redundancy.